packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -189,3 +189,7 @@ export const noIndexSetting = new PublicInstanceSetting<boolean>('noindex', fals
 export const verifyEmailsSetting = new PublicInstanceSetting<boolean>("verifyEmails", true, "optional");
 
 export const hasCuratedPostsSetting = new PublicInstanceSetting<boolean>("hasCuratedPosts", false, "optional");
+export const hasSideCommentsSetting = new PublicInstanceSetting<boolean>("comments.sideCommentsEnabled", isLWorAF, "optional");
+export const hasCommentsTableOfContentSetting = new PublicInstanceSetting<boolean>("comments.tableOfContentsEnabled", isLWorAF, "optional");
+export const hasDialoguesSetting = new PublicInstanceSetting<boolean>("dialogues.enabled", true, "optional");
+export const hasPostInlineReactionsSetting = new PublicInstanceSetting<boolean>("post.inlineReactionsEnabled", isLWorAF, "optional");
```

Comment:
Minor nitpick: maybe this should be `"posts.inlineReactionsEnabled"` instead of `"post.inlineReactionsEnabled"`, since "comments" and "dialogues" are pluralized?